extensions/linear/src/components/CreateIssueForm.tsx

```diff
@@ -198,6 +205,7 @@ export default function CreateIssueForm(props: CreateIssueFormProps) {
   const { cycles } = useCycles(values.teamId);
   const { issues } = useIssues(getLastCreatedIssues);
   const { projects } = useProjects(values.teamId);
+  const { milestones } = useMilestones(values.projectId);
```

Comment:
Let's use the `execute` boolean to make sure milestones are loaded only once a project is selected.

```suggestion
  const { milestones } = useMilestones(values.projectId, { execute: !!values.projectId} );
```